miniapps/solvers/block_fespace_operator.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef MFEM_BLOCK_FESPACE_OPERATOR
+#define MFEM_BLOCK_FESPACE_OPERATOR
+
+#include "mfem.hpp"
+
+namespace mfem
+{
+
+/// @brief Operator for block systems arising from different arbitrarily many finite element spaces.
+///
+///        This operator can be used with FormLinearSystem to impose boundary
+///        conditions for block systems arise from mixing many types of
+///        finite element spaces. Each block is intended to operate on
+///        L-Vectors. For example, a block may be a BilinearForm.
+class BlockFESpaceOperator : public Operator
+{
+private:
+   /// @brief Offsets for the square "A" operator.
```

Comment:
Very minor: don't need `@brief` here (and below) if the `///` comments are only one line long.

Reply:
Deleted the unnecessary `@brief`s